README.md

```diff
@@ -9,43 +9,46 @@ Although there exists a bunch of existing PBFT implementations, we decide to wri
 
 
 
-### Installation
+### Build
 
 #### Step 1
 
 Install [Docker](https://docs.docker.com/install/) and [HyperMake](http://evo-cloud.github.io/hmake/quickguide/install/).
 
-Make sure you have hmake in your `$GOBIN` path.
 
-### Step 2
+#### Step 2
 
+check hmake command options
 ```
-go get -u github.com/truechain/truechain-consensus-core
+git clone https://github.com/hixichen/truechain-consensus-core.git
+cd truechain-consensus-core
```

Comment:
I feel we should keep the `go get -u` way of importing the source tree into $GOPATH/src.